Underanalyzer/Decompiler/Warnings/DecompileDataLeftoverWarning.cs

```diff
@@ -1,13 +1,24 @@
-﻿namespace Underanalyzer.Decompiler;
+﻿/*
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+namespace Underanalyzer.Decompiler.Warnings;
 
 /// <summary>
 /// Represents a warning that occurs when data is left over on the VM stack at the end of a fragment.
-/// With default settings, this is not a warning, and is instead an exception.
 /// </summary>
+/// <remarks>With the default settings, this is not a warning, and is instead an exception.</remarks>
 public class DecompileDataLeftoverWarning : IDecompileWarning
 {
+    /// <inheritdoc/>
     public string Message => $"Data left over on VM stack at end of fragment ({NumberOfElements} elements).";
+    
+    /// <inheritdoc/>
     public string CodeEntryName { get; }
+    
+    // TODO: ??? How many elements are still left?
```

Comment:
Yeah, so this is how many elements are left over on the VM stack (at the end of a fragment).